CHANGELOG.md

```diff
@@ -22,6 +22,7 @@
 * The lifecycle of the sync client is now separated from the event loop/socket provider it uses for async I/O/timers. The sync client will wait for all outstanding callbacks/sockets to be closed during destruction. The SyncSocketProvider passed to the sync client must run until after the sync client is destroyed but does not need to be stopped as part of tearing down the sync client. ([PR #6276](https://github.com/realm/realm-core/pull/6276))
 * The default event loop will now keep running until it is explicitly stopped rather than until there are no more timers/IO to process. Previously there was a timer set for very far in the future to force the event loop to keep running. ([PR #6265](https://github.com/realm/realm-core/pull/6265))
 * Disable failing check in Metrics_TransactionTimings test ([PR #6206](https://github.com/realm/realm-core/pull/6206))
+* Update websocket error messages to be more descriptive ([PR #6298](https://github.com/realm/realm-core/pull/6298))
```

Comment:
I know this is an internal message but it's a bit misleading. It's actually just surfacing to the user the actual websocket/http error.